services/user.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str,
+    first_name: str,
+    last_name: str) -> User:
+    User = get_user_model()
+    user = User.objects.create(username="username")
```

Comment:
The username is being set as a string literal "username". It should use the `username` parameter instead: `username=username`.